### ISSUE TITLE
[attribute='value' i] case insensitivity is not correctly handled

### ISSUE THREAD

**Issue by pixelastic** [jakubpawlowicz/clean-css#902]
Seems like `clean-css` is too aggressively minifying attributes selectors that contains an ` i` to denote case-insensitivity.

### Input
```css
input[value="test" i] { color:#FFFFFF; }
```

### Output
```css
input[value=testi]{color:#fff}
```

### Expected output
```css
input[value='test' i]{color:#fff}
```

### Version
I'm using the commandline `cleancss` binary, installed through `yarn add global` on Ubuntu 16.04. The `cleancss` version is 4.0.7, with the default options.

### More information

[According to MDN](https://developer.mozilla.org/en/docs/Web/CSS/Attribute_selectors), one can use the `[attribute='value' i]` syntax to make the attribute check case-insensitive.
![image](https://cloud.githubusercontent.com/assets/283419/23438449/2d5c8764-fe12-11e6-98a9-bd856ef2587c.png)

**Comment by jakubpawlowicz** [comment]
This is a bug indeed. Fix is coming!

**Comment by jakubpawlowicz** [comment]
From what I've seen `input[value='test'i]{color:#fff}` (without space) works equally well in Chrome and Firefox.

**Comment by pixelastic** [comment]
Yep, works for me as well. Removing the space should work, as long as the quotes around the value are kept. 

This means the ideal solution in terms of number of chars used would be:
- For multi-words values, quotes should be kept but space removed (`[value='multi words'i]`)
- For single words values, quotes can be dropped, but space kept (`[value=word i]`)

**Comment by jakubpawlowicz** [comment]
Indeed! The fix is on 4.0 branch, due as 4.0.9 later today or tomorrow morning.

**Comment by jakubpawlowicz** [comment]
It's fixed in 4.0.9 out in npm registry now.
